modules/mobile/app/serializers/mobile/v0/user_serializer.rb

```diff
@@ -120,8 +120,8 @@ def authorized_services
 
       def direct_deposit_update_access?
         user.authorize(:ppiu, :access_update?)
-      rescue EVSS::PPIU::ServiceException => e
-        Rails.logger.error('Error fetching user data from EVSS', user_uuid: user.uuid, details: e.messages)
+      rescue => e
+        Rails.logger.error('Error fetching user data from EVSS', user_uuid: user.uuid, details: e.messages || e.message)
```

Comment:
the PPIU service Exception uses the plural `messages` but other errors use the singular so capturing both here.

Reply:
Good catch

Reply:
This will probably cause errors. I'd get the message in a separate line.
```
message = e.respond_to?(:messages) ? e.messages : e.message
```
Alternatively, you could catch the errors separately:
```
rescue EVSS::PPIU::ServiceException => e
  # whatever
rescue => e
  # also whatever
```